src/index.ts

```diff
@@ -4,7 +4,7 @@ import * as toolCache from '@actions/tool-cache';
 
 type Option<T> = T | undefined;
 
-const REQUIRED_NODEJS_MAJOR_VERSION = 14;
+const REQUIRED_NODEJS_MAJOR_VERSION = 18;
```

Comment:
Should this be 20?  Or we're saying that the min is 18, but it's compatible with up to 20?

Reply:
20 is what the GH Actions JS runs under, inside GH.
18 is what the CLI code runs under, on the system where the CLI will be installed.

These are disparate runtimes, @jbaldassari